README.md

```diff
@@ -19,6 +19,7 @@ FIREHOSE_AWS_REGION=us-west-2 \
 FIREHOSE_STREAM_NAME=firehose-test \
 LOG_FILE=/tmp/kcl_stderr \
 RATE_LIMIT=100 \
+DEPLOY_ENV="local" \
```

Comment:
The cases where I think we've agreed it's OK is when the app includes an infra library (kayvee), so from the app's perspective it's unaware of how those env vars are used.